Update black STM32F407VET6 board pinout to SPECTRE V0.5

Tjeerdie opened the pull request:
Updated the pinout for STM32F4 experimental board to V0.5 of the board.

TBAMax commented:
Hi.
Is there some important changes with pins? For example some hardware limitations why some pin must be changed from the previous arrangement. Can you comment on those if there are.

Tjeerdie commented:
Hi

Well just for routing of the board, mainly and adding injectors 5 to 8. I
left the standard board the same. I could add another board to the list but
to my knowledge there is no other open source board using this pinout.


On Sat, 18 Jul 2020, 12:28 Teemo Vaas, <notifications@github.com> wrote:

> Hi.
> Is there some important changes with pins? For example some hardware
> limitations why some pin must be changed from the previous arrangement. Can
> you comment on those if there are.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/noisymime/speeduino/pull/419#issuecomment-660462464>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH4RTSEMUA4EB7EYO437E63R4F2LLANCNFSM4O4ZXCAA>
> .
>

Tjeerdie commented:
Full story:

I added this board to speeduino.ini and code base 10 months ago for my then prototype STM32 based board. Also added the same pin out as (default) case, that is the speeduino 0.4 board. This design is now at version 0.5.1 and all looks to be working. Engine is running on this. See: https://github.com/Tjeerdie/SPECTRE 

Now I changed the board pin out to be correct with version 0.5 and 0.5.1 . And I left the  speeduino 0.4 board as before so that anyone using this pin out can select speeduino 0.4  board for testing/design. But I do not know of anyone using this specific pin out for a design yet.
